index.js

```diff
@@ -0,0 +1,125 @@
+const ALL_URL = "https://trektravel.herokuapp.com/trips";
+
+
+const reportStatus = (message) => {
+    const statusContainer = $('#status-message');
+    statusContainer.empty();
+    statusContainer.append(`<p>${message}</p>`);
+};
+
+
+// loads all trips for viewing
+const loadTrips = () => {
+    reportStatus('Loading trips...');
+
+    const tripList = $('#trip-list');
+    tripList.empty();
```

Comment:
It's great that you're emptying the list before appending new entries.  You should also empty the details section before adding content to the details section.